types/twine-sugarcube/test/state.ts

```diff
@@ -0,0 +1,76 @@
+function testStoryMoment(m: TwineSugarCube.StoryMoment): void {
+	const s: string = m.title;
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/twine-sugarcube/test/state.ts;
  Line: 2;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.